doc/sphinx/source/get-started/installation.rst

```diff
@@ -245,7 +245,7 @@ Installation from source on M1/M2 Macs
 --------------------------------------
 
 Installation on M1/M2 Macs directly with cmake is not directly supported.
-If you install following this tutorial PDFs will need to be evolved with ``evolven3fit_new``.
+If you install following this tutorial PDFs will need to be evolved with ``evolven3fit``.
```

Comment:
```suggestion
PDFs are evolved with ``evolven3fit``.
```

I guess this was a search and replace... which means there will be `evolven3fit <runfolder> <number of replicas>` still scattered around that should be `evolv3nfit evolve <runfolder`>

Reply:
You're right, I didn't check for mentions of the old `evolven3fit` in the docs